src/test/scala/is/hail/expr/ir/CompileSuite.scala

```diff
@@ -290,4 +290,47 @@ class CompileSuite {
     intercept[RuntimeException](run(Array(-1.0,null,1.0), 1))
   }
 
+  @Test
+  def getFieldSum() {
+    val tL = TStruct("0" -> TFloat64())
+    val scopeStruct = TStruct("foo" -> TInt32())
+    val tR = TStruct("x" -> TFloat64(), "scope" -> scopeStruct)
+    val ir = ApplyBinaryPrimOp(Add(),
+      GetField(In(0,tL),"0"),
+      GetField(In(1,tR),"x"))
+    val region = MemoryBuffer()
+    val loff = addStruct(region, "0", 3.0)
+    val roff = addStruct(region,
+      "x", 5.0,
```

Comment:
why new line here? and in next test

Reply:
to keep each field on one line; seems more readable to me.